src/cli/commands/dev.ts

```diff
@@ -89,33 +97,19 @@ export class Dev implements Command {
           }
 
           runDebouncedReflection(layout)
+
+          return {
+            entrypointScript: getTranspiledStartModule(layout),
+          }
```

Comment:
Makes sure the link options are updated with the new entrypoint script that potentially contains new/removed imports